docs/containers/debug-node.md

```diff
@@ -1,44 +1,22 @@
 ---
 Area: containers
 ContentId: F0C800DD-C477-492D-9545-745F570FE042
-PageTitle: Debug a Node.js app running in a Docker container
+PageTitle: Configure & troubleshoot debugging of Node.js apps running in a Docker container
 DateApproved: 12/12/2019
-MetaDescription: Debug a Node.js app running in a Docker container, using Visual Studio Code.
+MetaDescription: How to configure and troubleshoot debugging of Node.js apps running in a Docker container, using Visual Studio Code.
 ---
-# Debugging Node.js within Docker containers
 
-## Walkthrough
+# Common Node.js Debugging Configuration Tasks
 
-Start with an existing Node.js application or create a new one
+When adding Docker files to a Node.js project, tasks and launch configurations are added intended to enable one to debug that application within a Docker container. However, due to the large ecosystem surrounding Node.js, those tasks cannot accommodate every application framework or library, which means that some applications will require additional configuration.
```

Comment:
Are there missing words here between "added" and "intended"?

Reply:
`intended` was not intended. :-)